subcommand/server-acl-init/command_test.go

```diff
@@ -1233,6 +1211,11 @@ func mockReplicatedSetup(t *testing.T, bootToken string) (*fake.Clientset, *api.
 // replicatedSetup is a helper function for completeReplicatedSetup and
 // mockReplicatedSetup. If bootToken is empty, it will run the server-acl-init
 // command to set up replication. Otherwise it will do it through config.
+// Returns the Kubernetes client for the secondary DC,
+// a Consul API client initialized for the secondary DC,
+// ACL replication token, the address of the secondary Consul server, and a
```

Comment:
ordering is incorrect. It's address of secondary server, replication token

Reply:
Updated.